ci/build_cpp.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Update env vars
+source rapids-env-update
+
+# TODO: Move in recipe build?
+export CMAKE_GENERATOR=Ninja
+
+# TODO: Move to job config
+export CUDA=11.5
+
+# Check env
+source ci/check_env.sh
+
+################################################################################
+# BUILD - Conda package builds (LIBRMM)
+################################################################################
```

Comment:
We can remove all comment blocks like this. I don't think they provide much value.